test/fixtures/css-modules--false/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "css-modules--false",
+  "scripts": {
+    "build": "microbundle --no-sourcemap --css-modules false"
+  }
+}
```

Comment:
Minor nit, a lot of these files don't seem to have a final newline, does your editor not respect the editorconfig?

Reply:
Sorry my bad, I assumed prettier took care of that. Will update with new lines.

Reply:
Oh, do we have conflicting editorconfig and prettier configs?

Reply:
No sorry! Saw prettier and assumed it was used... But actually, its just running over the markdown files.

https://github.com/developit/microbundle/blob/master/package.json#L30-L31

Will fix these newline issues none the less.

Reply:
Here's a suggestion, why don't you approve this PR. Get it merged, and I'll raise a separate PR that runs prettier over all source files + add's in the husky pre-commit hook for that.

Reply:
Sure